addon/components/simple-typeahead/template.hbs

```diff
@@ -1,16 +1,25 @@
 <div class="input">
-  <input
+  {{input
     type='text'
-    value={{value}}
-    placeholder={{placeholder}}
-    oninput={{action 'input'}}
-    onkeydown={{action 'keydown'}}
-  />
+    spellcheck=false
+    value=value
+    placeholder=placeholder
+    input=(action 'input')
+    keyDown=(action 'keydown')
+  }}
 
-  <button title='clear input' class='clear' disabled={{valueIsEmpty}} {{action 'clear'}}>&#10005;</button>
+  {{#if showReset}}
+    <i title='clear input' class='clear' {{action 'clear'}}>
```

Comment:
This should be wrapped in a `<button>` element to be more semantic.

But rather than change from ASCII to svg, I'd like to just allow this to be easily customizable.  Have a thought how to do this that I need to try first.